NWHarvest.Web/Migrations/201711100017429_StateAndCountyModels.cs

```diff
@@ -0,0 +1,42 @@
+namespace NWHarvest.Web.Migrations
```

Comment:
Storing a single state and related counties might be overkill in db.  Propose creating WashingtonState class instead.

Reply:
I thought of that as well. But the db approach feels better for making the website open to users outside Washington. Although that will still require a change in how the County drop down gets updated. Let me know if you still feel we should go with the Class approach, and I will make that change.

Reply:
Yea, I prefer going with class approach using a StateViewModel that defaults to Washington state values.  This model can easily extended using another state by overriding the default values via in-memory or db.

I'm contemplating whether we need a State form field at all.  Since one of the requirements is that all users must be within Washington, we can implicitly add it when updating the db.  One less thing the user have to see when registering.

Reply:
Sure. I updated the migration. About the state form field, I agree it isn't needed as per current requirements and we should remove it. But let's do that in a separate pull request.